src/components/Garden/ModalFlows/ChallengeProposalScreens/ChallengeRequirements.js

```diff
@@ -57,6 +62,35 @@ function ChallengeRequirements({
 
   return (
     <div>
+      <Info
+        title="Challenge guidelines"
+        css={`
+          margin-bottom: ${2 * GU}px;
+        `}
+      >
+        Challenging a proposal invokes{' '}
```

Comment:
Reading this bit would make me think that by challenging the proposal, it would be immediately  raised to Celeste.

Reply:
I agree. We can state that needs to be raised to Celeste by the proposal creator.

Reply:
Great catch